kubernetes/linux/setup.sh

```diff
@@ -1,9 +1,27 @@
-TMPDIR="/opt"
-cd $TMPDIR
+cd /opt
+
+# set up apt repo for ruby2.6
+sudo apt-get install software-properties-common -y
+sudo apt-add-repository ppa:brightbox/ruby-ng -y
+# set up apt repo for fluent-bit(td-agent-bit)
+wget -qO - https://packages.fluentbit.io/fluentbit.key | sudo apt-key add -
+sudo echo "deb https://packages.fluentbit.io/ubuntu/xenial xenial main" >> /etc/apt/sources.list
+
+sudo apt-get update
+
+apt-get install -y libc-bin wget openssl curl sudo python-ctypes init-system-helpers  net-tools rsyslog cron vim dmidecode apt-transport-https gnupg
+
+# install telegraf in parallel with everything else to save build time
```

Comment:
do you know how much time this saves ?

Reply:
Quite a bit after the initial build. 

current ci_dev fresh build:
- docker build took 406.7s
- pushing docker image required uploading 595MB + 31MB + 60.5kB

current ci_dev rebuild after one line golang change:
- docker build took 398.8s
- pushing docker image required uploading 595MB + 31.5MB + 60.5kB

this PR fresh build:
- docker build took 373.4s
- pushing docker image required uploading 645MB + 33.4kB + 31.5MB + 49.6MB

this PR rebuild after one line golang change:
- docker build took 5.7s
- pushing docker image required uploading 31.5MB + 49.6MB

(make and the bundler still run on every build, so builds are longer than my posted times. I didn't include them because they aren't changed by this PR)